### DISCUSSION TITLE
Inclusion of wave attenuation by vegetation: source code

### DISCUSSION THREAD

**Discussion by scausio** [NOAA-EMC/WW3#917] [Q&A]
To whom it may concern.
I read a very interesting paper about the inclusion of the vegetation dissipative term in WW3.

 Abdolali A, Hesser TJ, Anderson Bryant M, Roland A, Khalid A, Smith J, Ferreira C, Mehra A and Sikiric MD (2022) Wave Attenuation by Vegetation: Model Implementation and Validation Study. Front. Built Environ. 8:891612. doi: 10.3389/fbuil.2022.891612

I found there that "the names of the repository/repositories and accession number(s) can be found at https://github.com/NOAA-emc/ww3".
However, I was not able to find the code. Can anyone provide me some info more, or  is the code not available yet?  
Kind regards,
Salvatore

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali @thesser1 @maryanderson64  @aronroland  can you please let @scausio know the status of this being added back to the WW3 develop branch?

**Comment by aronroland** [comment]
Ciao Salvatore, 

my understanding is that the new capability will be available in the short to midterm. We r working on the extension of the shallow water physics in the develop branch with a quite decent pace. If u need more details please contact @thesser1 or @aliabdolali.

Cheers 

Aron
